docs/DeveloperGuide.md

```diff
@@ -117,22 +117,51 @@ The `Model`,
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2021S1-CS2103-T14-3/tp/blob/master/src/main/java/seedu/address/storage/Storage.java)
 
 The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
-* can save the address book data in json format and read it back.
+* can save the travel planner data in json format and read it back.
 
 ### Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `seedu.address.commons` package.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Find Feature
+
+#### Implementation
+
+Current implementation of Find feature is facilitated by `NameContainsKeywordsPredicate` which implements 
+`Predicate<Nameable>` and has the **test** method overriden with a custom implementations.
+ 
+The `NameContainsKeywordsPredicate#test(Nameable)` is as follows:  
+* loops through the **keywords** (of type `List<String>`) and return true if there is any match with the `Nameable`
```

Comment:
Not sure if there should be a full stop here.

Reply:
Should it be:
* loops through the **keywords** (of type `List<String>`) and return true if there is any match with `Nameable.getName().name`